### PR TITLE
[oneTBB] Fix namespaces and headers

### PR DESCRIPTION
**Pull request by alexandraepan**
This patch replaces tbb namespaces and headers with oneapi tbb.  

Signed-off-by: Alexandra Epanchinzeva alexandra.epanchinzeva@intel.com

### CODE REVIEW

File: source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst [hunk 1]
```diff
@@ -13,39 +13,42 @@ A ``blocked_range3d`` is the three-dimensional extension of ``blocked_range2d``.
...
-            bool empty() const;
-
-            // Access
-            bool is_divisible() const;
-            const page_range_type& pages() const;
-            const row_range_type& rows() const;
-            const col_range_type& cols() const;
-        };
-    }
+    namespace oneapi{
```
Comment (omalyshe) [inline source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst:16]: Space is missed before {

File: source/elements/oneTBB/source/flow_graph/async_node_cls.rst [hunk 2]
```diff
@@ -12,8 +12,9 @@ the user or another runtime.
 
 .. code:: cpp
 
-    // Defined in header <tbb/flow_graph.h>
+    // Defined in header <oneapi/tbb/flow_graph.h>
 
+    namespace oneapi {
     namespace tbb {
```
Comment (omalyshe) [inline source/elements/oneTBB/source/flow_graph/async_node_cls.rst:18]: Shouldn't we use indentation for nested namespaces? Some pages use it (see e.g. сontainers/concurrent_vector_cls.rst 
@anton-potapov What do you think?

Comment (omalyshe) [reply source/elements/oneTBB/source/flow_graph/async_node_cls.rst:18]: Discussed internally and decided to go as-is with this patch and fix indentation later.

File: source/elements/oneTBB/source/info_namespace.rst [hunk 3]
```diff
@@ -13,13 +13,15 @@ Interfaces to query information about execution environment.
 
    // Declared in info.h
```
Comment (anton-potapov) [inline source/elements/oneTBB/source/info_namespace.rst:14]: I believe here should be a full path to the header :  `<oneapi/tbb/info.h>`

**Review by anton-potapov — COMMENTED** [review]
Please update dates in the license header of affected files

File: source/elements/oneTBB/source/task_scheduler/task_arena/task_scheduler_observer_cls.rst [hunk 4]
```diff
@@ -106,15 +108,15 @@ The following example sketches the code of an observer that pins oneTBB worker t
 
 .. code:: cpp
 
-    class pinning_observer : public tbb::task_scheduler_observer {
+    class pinning_observer : public oneapi/tbb::task_scheduler_observer {
```
Comment (anton-potapov) [inline source/elements/oneTBB/source/task_scheduler/task_arena/task_scheduler_observer_cls.rst:111]:
```suggestion
    class pinning_observer : public oneapi::tbb::task_scheduler_observer {
```

**Review by anton-potapov — COMMENTED** [review]
Please update the PR to the latest main branch (in order to resolve to conflicts)

File: source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst (hunk 1 above)
Comment (anton-potapov) [reply source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst:16]:
```suggestion
    namespace oneapi {
```

File: source/elements/oneTBB/source/containers/concurrent_vector_cls/deduction_guides.rst [hunk 5]
```diff
@@ -39,18 +39,24 @@ This deduction guide only participate in the overload resolution if the followin
...
         std::array<int, 100> arr;
 
-        // Deduces cv1 as tbb::concurrent_vector<int>
-        tbb::concurrent_vector cv1(arr.begin(), arr.end());
+        // Deduces cv1 as oneapi::tbb::concurrent_vector<int>
+        oneapi::tbb::concurrent_vector cv1(arr.begin(), arr.end());
 
         std::allocator<int> alloc;
 
+<<<<<<< HEAD
```
Comment (omalyshe) [inline source/elements/oneTBB/source/containers/concurrent_vector_cls/deduction_guides.rst:54]: Incorrectly merged

Comment (anton-potapov) [reply source/elements/oneTBB/source/containers/concurrent_vector_cls/deduction_guides.rst:54]: good catch

Comment (anton-potapov) [reply source/elements/oneTBB/source/containers/concurrent_vector_cls/deduction_guides.rst]: fixed

File: source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst (hunk 1 above)
Comment (anton-potapov) [reply source/elements/oneTBB/source/algorithms/blocked_ranges/blocked_range3d_cls.rst]: fixed

File: source/elements/oneTBB/source/info_namespace.rst (hunk 3 above)
Comment (anton-potapov) [reply source/elements/oneTBB/source/info_namespace.rst]: fixed

**Pull request merged**